Add BBEs related to the data.yaml module (#5531)
ballerina-platform/ballerina-distribution: examples/yaml-to-anydata/yaml_to_anydata.md

# YAML to anydata conversion

> The `data.yaml` library provides multiple APIs to perform the conversion from YAML source, which can be provided as a `string`, `byte array`, or `byte block stream`, 

Reviewer: Fix backticks in `string`, `byte array`, or `byte block stream`,

# YAML to anydata conversion

> The `data.yaml` library provides multiple APIs to parse YAML source, in the form of a string, byte array, or byte block stream, into a subtype of Ballerina `anydata`.

Reviewer: ```suggestion
The `data.yaml` library provides multiple functions to parse YAML source, in the form of a string, byte array, or byte block stream, as a value that belongs to a subtype of `anydata`.
```
